R/create_extendr_package.R

```diff
@@ -0,0 +1,55 @@
+
+#' Create a project for R package development with Rust
+#'
+#' @description
+#' This function creates an R project directory for package development
+#' with Rust extensions.
+#'
+#' @param path a path to new directory
+#' @param ... arguments passed on to `usethis::create_package()` and
+#' `rextendr::use_extendr()`
+#'
+#' @return Path to the newly created project or package, invisibly.
+#' @keywords internal
+#'
+#' @noRd
```

Comment:
Is there any reason why we wouldn't export this function? Seems like a handy function to be exported.

Reply:
I believe that's a question for you and @Ilia-Kosenkov to decide on.

Reply:
I'd vote for @export here to include it in the namespace. But that's not necessary for this PR.